addons/website_sale/tests/test_address.py

```diff
@@ -12,6 +12,7 @@
 from odoo.addons.website.tools import MockRequest
 from odoo.addons.website_sale.controllers.main import WebsiteSale
 from odoo.addons.website_sale.tests.common import WebsiteSaleCommon
+import json
```

Comment:
Should be first in the imports (https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports)